### ISSUE TITLE
Scripts like qgis_setup.sh are missing

### ISSUE THREAD

**Issue by reinout** [qgis/qgis-docker#114]
We use the docker images for testing our plugins, using `qgis_setup.sh` like documented here: https://github.com/qgis/QGIS/tree/master/.docker#running-tests-for-a-python-plugin

But... the `Dockerfile` in there isn't used anymore as far as I can see and it has been replaced by the one in this qgis-docker repository. And the install method here misses the test setup scripts provided by the older docker images:

    $ docker run --rm -it docker.io/qgis/qgis:release-3_16 which qgis_setup.sh
    /usr/bin/qgis_setup.sh
    $ docker run --rm -it docker.io/qgis/qgis:3.38 qgis_setup.sh
    docker: Error response from daemon .... : "qgis_setup.sh": executable file not found in $PATH: unknown.

So: is this a bug? Or intended behaviour? Is there a different docker I should be using?
See related issue in the qgis repo: https://github.com/qgis/QGIS/issues/57931

**Comment by 3nids** [comment]
scripts from https://github.com/qgis/QGIS/tree/master/.docker/qgis_resources/test_runner should be copied and adapted

**Comment by reinout** [comment]
Would it be OK to copy/adapt them into this repository? Or should we do it locally into our test setup?
Including them in this repository seems logical to me, but perhaps the goal is to keep this repository as clean as possible, that's why I'm asking :-)

**Comment by 3nids** [comment]
Yes that would be great to integrate them here if that helps at testing plugins.
